FastBank.Infrastructure/Repository/TransactionRepository.cs

```diff
@@ -56,5 +82,51 @@ public void AddTransactionsReport(TransactionsReport transactionsReport)
         {
             _repository.Add<TransactionsReportDTO>(new TransactionsReportDTO(transactionsReport));
         }
+
+        public List<TransactionsReport> GetTransactionsReports(int currentPage = 1)
```

Comment:
The class name is not ok. Should be TransactionReport not TransactionReports. TransactionReports for me means that it holds in a property a list of TransactionReport, but this is not true. Also method name should be changed accordingly.

Reply:
Also check and rename the variables in that method

Reply:
Not agree with comments. The class is for Report which can have few or more transactions